.github/workflows/build.yml

```diff
@@ -45,3 +46,4 @@ jobs:
           WEBHOOK_SECRET: ${{ secrets.WEBHOOK_SECRET }}
           APP_ID: ${{ secrets.APP_ID }}
           PRIVATE_KEY: ${{ secrets.PRIVATE_KEY }}
+          CLOUDFLARE_ACCOUNT_ID: ${{ secrets.CLOUDFLARE_ACCOUNT_ID }}
```

Comment:
I think it should be in `with`, not env (https://github.com/cloudflare/wrangler-action?tab=readme-ov-file#deploy-your-pages-site-production--preview)

Reply:
Interesting, according to wrangler documentation itself it is an environment variable
https://developers.cloudflare.com/workers/wrangler/configuration/#inheritable-keys
Will change to `with`, thanks for noticing.

Reply:
maybe it would also work but just to be on the safe side :)